scripts/debug_gemini.sh

```
@@ -0,0 +1,105 @@
+echo "1. 環境情報"
+echo "- OS: $(uname -a)"
+echo "- Container ID: $(hostname)"
+echo "- Current Dir: $(pwd)"
+echo "- User: $(whoami)"
+echo
+
+# Gemini CLI確認
+echo "2. Gemini CLI バイナリ確認"
+echo "- パス: $(which gemini || echo 'NOT FOUND')"
+echo "- バージョン: $(gemini --version 2>/dev/null || echo 'バージョン取得失敗')"
+echo "- ファイル情報: $(ls -la $(which gemini) 2>/dev/null || echo 'ファイル情報取得失敗')"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)